recipes/very-simple-smtps/all/conanfile.py

```diff
@@ -0,0 +1,171 @@
+        # src_folder must use the same source folder name the project
+        basic_layout(self, src_folder="src")
+
+    def requirements(self):
+        for package in self.required_packages:
+            self.requires(package)
+
+    def validate(self):
+        # validate the minimum cpp standard supported. For C++ projects only
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+        check_min_vs(self, 191)
+        if not is_msvc(self):
+            minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+            if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+                )
+        # in case it does not work in another configuration, it should validated here too
+        if is_msvc(self) and self.info.options.shared:
+            raise ConanInvalidConfiguration(f"{self.ref} can not be built as shared on Visual Studio and msvc.")
+
+    # if another tool than the compiler or Meson is required to build the project (pkgconf, bison, flex etc)
+    def build_requirements(self):
+        # CCI policy assumes that Meson may not be installed on consumers machine
+        self.tool_requires("meson/1.2.0")
+        # pkgconf is largely used by Meson, it should be added in build requirement when there are dependencies
+        if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
+            self.tool_requires("pkgconf/1.9.3")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        # default_library and b_staticpic are automatically parsed when self.options.shared and self.options.fpic exist
+        # buildtype is automatically parsed for self.settings
+        tc = MesonToolchain(self)
+        # In case need to pass definitions directly to the compiler
+        # tc.preprocessor_definitions["MYDEFINE"] = "MYDEF_VALUE"
+        # Meson project options may vary their types
+        # tc.project_options["tests"] = False
+        tc.generate()
+        # In case there are dependencies listed on requirements, PkgConfigDeps should be used
+        tc = PkgConfigDeps(self)
+        tc.generate()
+        # In case there are dependencies listed on build_requirements, VirtualBuildEnv should be used
+        tc = VirtualBuildEnv(self)
+        tc.generate()
+
+    def _patch_sources(self):
+        apply_conandata_patches(self)
+        # remove bundled xxhash
+        # rm(self, "whateer.*", os.path.join(self.source_folder, "lib"))
+        # replace_in_file(self, os.path.join(self.source_folder, "meson.build"), "...", "")
+
+    def build(self):
+        self._patch_sources()  # It can be apply_conandata_patches(self) only in case no more patches are needed
```

Comment:
```suggestion
    def build(self):
```
No patches to be applied. Let's keep it simpler, please.